sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.institutions.dto.ts

```diff
@@ -85,7 +87,7 @@ export class CompletedOfferingChangePaginationOptionsAPIInDTO extends Pagination
  */
 export class InProgressOfferingChangePaginationOptionsAPIInDTO extends PaginationOptionsAPIInDTO {
   @IsOptional()
-  @IsIn(["applicationNumber", "fullName"])
+  @IsIn(["applicationNumber", "fullName", "dateSubmitted", "status"])
```

Comment:
IMO, I think we should create 2 DTO for each controller and the sort filed is different for both

Reply:
ok. Created a separate DTO. Thanks!

Reply:
do we still need `"dateSubmitted", "status"]` in institution dto?

Reply:
Removed. Thanks!